icebergs_framework.F90

```diff
@@ -2871,9 +2887,10 @@ integer function berg_chksum(berg )
   itmp(37)=berg%start_year !Changed from 29 to 37 by Alon
   itmp(38)=berg%ine !Changed from 30 to 38 by Alon
   itmp(39)=berg%jne !Changed from 31 to 39 by Alon
+  itmp(40)=berg%iceberg_num !added  by Alon
```

Comment:
`itmp` needs to be declared `itmp(36+4)` in order to avoid an out-of-bounds at this line.